source/documentation/using_the_api/api_reference.md

```diff
@@ -338,34 +340,7 @@ Example response:
 }
 ```
 
-### <a name="downloadreg"></a>`GET /download-register`
-
-Download the full contents of the register in a ZIP file.
-
-*Note: This will download every entry and item as an individual JSON file. If you only want to download records, use `GET /records`.*
-
-Example URL: `https://local-authority-eng.register.gov.uk/download-register`
-
-Example request: `curl -o localauthorityeng.zip --request GET --url https://local-authority-eng.register.gov.uk/download-register`
-
-## Getting updates
-
-You can download the new entries in 2 ways:  
-
-* download another copy of the full register
-* download individual updated entries for the records you are using
-
-### `GET /register`
-
-You can find the latest entry number by looking at the register information and comparing the most recent entry number with your own copy.
-
-Example request: `https://local-authority-eng.register.gov.uk/register`
-
-### <a name="downloadreg"></a>`GET /download-register`
-
-See [endpoint for downloading register](#downloadreg).
-
-### `GET /record/{field-value}/entries`
+### <a name="entries">`GET /record/{field-value}/entries`
```

Comment:
I'd probably use `<a name="record-entries">` to distinguish it from the `/entries` endpoint.